### ISSUE TITLE
PhillipsMalickEtAl2023 needs pages

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [pacific-hake/hake-assessment#1003]
No pages and maybe wrong issue but the information was not available at the time of the 2023 assessment.
https://onlinelibrary.wiley.com/doi/full/10.1111/fog.12628

**Comment by aaronmberger-nwfsc** [comment]
The pdf says: Fisheries Oceanography.2023;1–13.  Maybe that is a placeholder?

**Comment by kellijohnson-NOAA** [comment]
I think so, I think that is just because it is 13 pages long.

**Comment by andrew-edwards** [comment]
Yes. So technically 'in press'? Can't guarantee the year something will be published in a proper volume (though presumably it would be 2023 still in this case).

**Comment by aaronmberger-nwfsc** [comment]
No, it is published and out and has a doi and everything.  It's just many journals new way of getting researchers stuff out the door before actual, physically journal issues are completed.  And in some cases, journals are only online (no more physical print).
